src/index.js

```diff
@@ -89,6 +89,14 @@ export default function commonjs ( options = {} ) {
 						const match = exportsPattern.exec( flattened.keypath );
 						if ( !match || flattened.keypath === 'exports' ) return;
 
+						if ( flattened.keypath === 'module.exports' && node.right.type === 'ObjectExpression' ) {
+							return node.right.properties.forEach( prop => {
+								if ( prop.computed ) return;
+								const name = prop.key.name;
+								if ( name === makeLegalIdentifier( name ) ) namedExports[ name ] = true;
```

Comment:
ah, right – I thought the `computed` check would be enough but you can do things like `[a]: a`. Fixed